04-spatial-operations.Rmd

```diff
@@ -168,7 +177,7 @@ typeof(sel_matrix)
 dim(sel_matrix)
 ```
 
-
+<!-- What do you think about placing this section before the spatial subsetting section since it sets the scene for subsetting, join, etc. -->
```

Comment:
What do you think about placing the topological section before the spatial subsetting section since it sets the scene for subsetting, join, etc.